source/MaterialXRuntime/Private/PvtNode.cpp

```diff
@@ -40,6 +30,27 @@ PvtNode::PvtNode(const RtToken& name) :
 {
 }
 
+void PvtNode::postConstructor()
```

Comment:
Post constructor method needed, because value allocation cannot be done in the main constructor, it must be done after the node object is initialized as a sharable RtObject. (this->getObject() needs to be called when creating values for the node)